### PR TITLE
[Lint](clean): Remove some warnings

### PR DESCRIPTION
**Pull request by alexandre-merle**
- Remove some warnings in lib/ files

### CODE REVIEW

File: lib/auth/AuthInfo.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default class AuthInfo {
     }
     // Check whether requester is an IAM user versus an account
     isRequesterAnIAMUser() {
-        return this.IAMdisplayName ? true : false;
+        return !!this.IAMdisplayName;
```
Comment (rahulreddy) [inline lib/auth/AuthInfo.js]: Did this actually throw a linter warning? If not, please use the older code as it's more clear and explicit.

File: lib/data/in_memory/backend.js [hunk 2]
```diff
@@ -12,7 +12,9 @@ export function resetCount() {
 export const backend = {
     put: function putMem(request, keyContext, reqUids, callback) {
         let value = new Buffer(0);
-        request.on('data', data => value = Buffer.concat([value, data]))
+        request.on('data', data => {
+            value = Buffer.concat([value, data]);
+        })
```
Comment (rahulreddy) [inline lib/data/in_memory/backend.js]: missing semi-colon

File: lib/auth/AuthInfo.js (hunk 1 above)
Comment (alexandre-merle) [inline lib/auth/AuthInfo.js]: Yes, it's a lint warning ^^

File: lib/data/in_memory/backend.js (hunk 2 above)
Comment (alexandre-merle) [inline lib/data/in_memory/backend.js]: The call continue below :)

Comment (rahulreddy) [inline lib/data/in_memory/backend.js]: my bad, I didn't notice that

File: lib/auth/AuthInfo.js (hunk 1 above)
Comment (rahulreddy) [inline lib/auth/AuthInfo.js]: Weird, I hate that warning

**Comment by rahulreddy** [comment]
👍

File: lib/auth/AuthInfo.js (hunk 1 above)
Comment (alexandre-merle) [inline lib/auth/AuthInfo.js]: yes, an ternary with yes / no will result in a warning

**Comment by LaurenSpiegel** [comment]
👍

**Pull request closed without merging**